lib/rules/jsx-no-literals.js

```diff
@@ -120,7 +120,12 @@ module.exports = {
       },
 
       JSXAttribute(node) {
-        const isNodeValueString = node.value && node.value && node.value.type === 'Literal' && typeof node.value.value === 'string';
+        let isNodeValueString;
+        if (context && context.options[0] && context.options[0].allowedStrings !== undefined) {
```

Comment:
why is this check needed? `config.allowedStrings` should already be an array containing either the default, or the provided value - what's the need to determine what the user actually passed?

Reply:
Thanks for catching that. Fixed & pushed changes